eZ/Bundle/EzPublishCoreBundle/Command/CleanupVersionsCommand.php

```diff
@@ -157,16 +167,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
                     $versionsCount
                 ), Output::VERBOSITY_VERBOSE);
 
-                $removeAll = $status === self::VERSION_ALL;
-                $removeDrafts = $status === self::VERSION_DRAFT;
-                $removeArchived = $status === self::VERSION_ARCHIVED;
-
                 $versions = array_slice(
                     array_filter($versions, function ($version) use ($removeAll, $removeDrafts, $removeArchived) {
                         if (
-                            ($removeAll && $version->status !== VersionInfo::STATUS_PUBLISHED) ||
-                            ($removeDrafts && $version->status === VersionInfo::STATUS_DRAFT) ||
-                            ($removeArchived && $version->status === VersionInfo::STATUS_ARCHIVED)
+                            ($removeAll && $version->status !== self::VERSION_STATUS[self::VERSION_PUBLISHED]) ||
+                            ($removeDrafts && $version->status === self::VERSION_STATUS[self::VERSION_DRAFT]) ||
+                            ($removeArchived && $version->status === self::VERSION_STATUS[self::VERSION_ARCHIVED])
```

Comment:
hmmm

Reply:
Fair. Reverted that https://github.com/ezsystems/ezpublish-kernel/pull/2431/commits/3eab4993291d07cac4b574e70574b5ba79e5bcc9 🙂